### DISCUSSION TITLE
Using JPEGView mylist.txt (slideshow) across dirs and drives

### DISCUSSION THREAD

**Discussion by slovuj** [sylikc/jpegview#170] [General]
It looks like the mylist.txt parameter on the JPEGview command line only looks 
for files in the current dir? If I craft a list.txt file like this, with explicit full paths:

D:\Knights.jpg
"C:\20130223.bmp"
C:\de\20150429_185649_HDR.jpg
C:\de\20150501_092308_HDR.jpg
D:\CYMERA_20160808_152626.jpg
E:\test\eva.PNG
D:\DSC_0775.JPG
"E:\CA\Vancouver 1.png"
"D:\examples\python 1.jpg"
"D:\examples\python 2.jpg"

and run JPEGView from a random working directory, say, C:\testing_jpegview:

`C:\JPEGView\JPEGView.exe" C:\TEMP\mylist.txt`

It tells me "The directory C:\testing_jpegview does not contain
any image files!" Of course it does not, but I gave you
a list to read line by line and display images named there.

I hoped that adding /slideshow would help, but no.

Am I doing something wrong? If not, please improve this feature,
which allows one to generate a list of image files across dirs,
local and network drives...

The `readme.html` states that the Slide Show Text File...
_All text formats must contain one file name per line. 
The file names must use paths relative to the folder 
of the slide show text file or absolute paths and must
 be terminated either with a hyphen ("), with a newline 
 character or with a < character.
The following examples show possible line formats:
d:\Images\test.jpg
test.jpg
"d:\My Documents\test.jpg"
<image>d:\My Documents\test2.jpg</image>_

What am I doing wrong?

**Comment by sylikc** [comment]
I'm going to construct some tests next week to test this behavior... I don't use the slideshow or pre-loaded file lists a whole lot, but let me recreate a test case with multiple directory structures all over the place... if there's a bug it should be relatively easy to find.
